Makefile

```diff
@@ -110,7 +107,7 @@ build-deps: clone-deps
 	(cd ../torchtext   && python setup.py clean && python setup.py develop)
 	(cd ../torchaudio  && python setup.py clean && python setup.py develop)
 	(cd ../detectron2  && python setup.py clean && python setup.py develop)
-	(cd ../functorch   && python setup.py clean && python setup.py develop)
+	(cd ../pytorch/functorch   && python setup.py clean && python setup.py develop)
```

Comment:
So people still need to install functorch even though it's back in core?

Reply:
we don't have consolidated build yet, so yes you still have to explicitly build it